ui/app/components/app/modals/modal.js

```diff
@@ -174,6 +176,30 @@ const MODALS = {
     ...accountModalStyle,
   },
 
+  EXTERNAL_SIGN: {
```

Comment:
anything called something like "external sign(er)" should be renamed something like "BidirectionalQrSign(er)"